src/libcrun/linux.c

```diff
@@ -1840,6 +1840,38 @@ make_parent_mount_private (const char *rootfs, libcrun_error_t *err)
   return 0;
 }
 
+// libcrun_create_kvm_device: explicitly adds kvm device.
+int
+libcrun_create_kvm_device (libcrun_container_t *container, libcrun_error_t *err)
```

Comment:
should we check if the device is not already specified in the OCI configuration?

Reply:
@giuseppe Yes that would be good will short the function early.